### ISSUE TITLE
caching issues

### ISSUE THREAD

**Issue by mguillau42** [mde/ejs#298]
Hi,
I recently updated ejs from 2.5.6 to 2.5.7 and caching does not seems to be working as I expect it to anymore.

I have enabled caching in my app.js
```js
...
app.set('view engine', 'ejs');
app.enable('view cache');
...
```
In the 2.5.6 version, if I were to modify a `.ejs` file while serving my application, it would render the original content. 
The modifications are live in the 2.5.7 version as if caching is not enabled.

Can you confirm that this is a bug ?

**Comment by User4martin** [comment]
Looks like you are using express? 

Express may be using it's own cache. Did your update also update express? Does the issues happen outside of express?

**Comment by mguillau42** [comment]
indeed, I am using express 4.15.4
I am not sure this has to do with express, since I can get caching to work by reinstalling ejs 2.5.6.
Even with express's recent update

**Comment by mde** [comment]
I can repro. I'll take a look.

**Comment by mde** [comment]
Apparently Express has moved stuff around again. We had made a special case in our security update for Express caching, and because we have to rely on internal APIs, when they make changes it breaks our special casing. (So it was your combo of newer Express + newer EJS.) Fixed in master in 8c575c98. Will be in the next release.
